main/evolution-data-server/template.py

```diff
@@ -1,10 +1,9 @@
 pkgname = "evolution-data-server"
-pkgver = "3.50.4"
+pkgver = "3.52.0"
 pkgrel = 0
 build_style = "cmake"
 # TODO: libgdata
```

Comment:
the todo itself also isn't needed

Reply:
(this was packaged at 3.42, in 3.46 gdata was removed in favor of internal stuff tho)